bedrock/firefox/templates/firefox/whatsnew/whatsnew-fx95-en.html

```diff
@@ -36,10 +35,7 @@ <h1 class="wnp-main-title">The results are in! These are the best articles of 20
       <p class="wnp-main-tagline">
         The long reads we all couldn’t put down. The most popular tech articles on the web. The great pieces of advice that we needed to hear. Dive into the most interesting articles of the year in Pocket’s Best of 2021.
       </p>
-
-      <div class="wnp-main-cta">
-        {{ pocket_fxa_button(entrypoint=entrypoint, button_text='See the winners', optional_parameters={'s': entrypoint, 'utm_campaign': utm_campaign, 'utm_content': 'signup-fxa'}, optional_attributes={'data-cta-text': 'Sign up and read now', 'data-cta-type': 'button'}) }}
-      </div>
+      <a href="https://getpocket.com/collections" class="wnp-main-cta mzp-c-button mzp-t-product">See the Winners</a>
```

Comment:
It works now but curious if we need to keep the `pocket_fxa_button` attribute. @alexgibson what do you think here?

Reply:
I'm not quite sure - alex would know the best!

Reply:
My hunch is that linking directly to https://getpocket.com/collections will not convert new Pocket signups, since you don't need to log-in to see / read the collections.

I'll defer to @slightlyoffbeat here, but I would suggest sticking to using `pocket_fxa_button()` to get people directly into the signup flow. But if signups aren't the goal here, then maybe this is fine.

Reply:
If we do stick with this URL, then let's make sure to add utm params and data attributes for GA:

```suggestion
      <a href="https://getpocket.com/collections?{{ utm_params }}" class="wnp-main-cta mzp-c-button mzp-t-product" data-cta-type="button" data-cta-text="See the Winners">See the Winners</a>
```

Reply:
@nathan-barrett @alexgibson the studio and PMM team decided to link to collections and are aware that this will not convert as well as the pocket fxa button. The correct CTA is getpocket.com/collections

Reply:
The campaign for pocket best of is:

english: utm_campaign=best-of-us
german: utm_campaign=best-of-de